yorkie/rpc/server.go

```diff
@@ -23,6 +23,7 @@ import (
 	gotime "time"
 
 	grpcmiddleware "github.com/grpc-ecosystem/go-grpc-middleware"
+	grpc_recovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
```

Comment:
```suggestion
	grpcrecovery "github.com/grpc-ecosystem/go-grpc-middleware/recovery"
```

Reply:
It would be nice to put this when we release our agent as a product.
But for now, It would be good to explicitly handle the error.

Reply:
That's right. So what do you think about adding an option to grpc config deciding whether to add grpc recovery interceptor or not?

Reply:
How about removing it from this PR and handling it as a separate issue?